WPXMLRPC/WPXMLRPCDataCleaner.m

```diff
@@ -82,18 +84,18 @@ @implementation WPXMLRPCDataCleaner (CleaningSteps)
 - (NSData *)cleanInvalidUTF8:(NSData *)data {
     NSData *result;
     iconv_t cd = iconv_open("UTF-8", "UTF-8"); // convert to UTF-8 from UTF-8
-	int one = 1;
-	iconvctl(cd, ICONV_SET_DISCARD_ILSEQ, &one); // discard invalid characters
-	
-	size_t inbytesleft, outbytesleft;
-	inbytesleft = outbytesleft = data.length;
-
-	char *inbuf  = (char *)data.bytes;
-	char *outbuf = malloc(sizeof(char) * data.length);
-	char *outptr = outbuf;
-
-	if (iconv(cd, &inbuf, &inbytesleft, &outptr, &outbytesleft) == (size_t)-1) {
-		// Failed iconv, possible errors to encounter in `errno`:
+    int one = 1;
+    iconvctl(cd, ICONV_SET_DISCARD_ILSEQ, &one); // discard invalid characters
+
+    size_t inbytesleft, outbytesleft;
+    inbytesleft = outbytesleft = data.length;
+
+    char *inbuf  = (char *)data.bytes;
+    char *outbuf = malloc(sizeof(char) * data.length);
+    char *outptr = outbuf;
+
+    if (iconv(cd, &inbuf, &inbytesleft, &outptr, &outbytesleft) == (size_t)-1) {
+        // Failed iconv, possible errors to encounter in `errno`:
```

Comment:
An unintentional change.  I think Xcode took care of the formatting due to some copy/pasting.  Since its switching tabs to spaces its actually more inline with our coding standards.  Call it a happy accident. :)